src/form-engine.component.tsx

```diff
@@ -118,7 +118,8 @@ const FormEngine: React.FC<FormProps> = ({
   }, [patient?.id, sessionMode, workspaceLayout]);
 
   const showButtonSet = useMemo(() => {
-    return sessionMode !== 'embedded-view' && workspaceLayout === 'minimized' && scrollablePages.size <= 1;
+    if (sessionMode == 'embedded-view') return false;
+    return workspaceLayout === 'minimized' || ('maximized' && scrollablePages.size <= 1);
```

Comment:
```suggestion
    if (sessionMode === 'embedded-view') {
      return false;
    }
    
    return workspaceLayout === 'minimized' || (workspaceLayout === 'maximized' && scrollablePages.size <= 1);
```

Reply:
Do the updated changes work in Form Render Test as well? I've tested that they work in the Patient Chart:

![CleanShot 2024-05-02 at 12  20 23@2x](https://github.com/openmrs/openmrs-form-engine-lib/assets/8509731/c20af34e-82c0-46d8-af83-cacc71a8aad0)

Reply:
Thanks @denniskigen, works just fine.